docs/source/onboarding_resources/computing/index.rst

```diff
@@ -280,22 +280,166 @@ lines from your trainings!
 Your Bash Configuration Files 
 -----------------------------
 
-Bash files contain commands you would enter into a command line, but 
-specifically ones you will use very often. For example, every time you 
-want to open a Jupyter session, the cluster needs certain information 
-and requirements. To find this information, it looks in your Bash files 
-rather than asking you to enter the same information every time. 
+Bash configuration files contain information and commands that are used when 
+interacting with the cluster. When the cluster tries to execute some command 
+line prompts, it will look to your bash configuration files for information 
+or filepaths. 
 
-However, Bash files can be confusing since it is less obvious when the 
-information is being used or what it is used for. Therefore, we have 
-provided a copy-and-paste formatting for information to be added to 
-your Bash files. 
+You should have 2 bash configuration files, your bash profile or :code:`.bash_profile` 
+and your bash rc or :code:`.bashrc`. 
```

Comment:
It might be nice to mention that the "rc" in `.bashrc` stands for "run commands" and originates from the predecessors of Unix (CTSS and Multics), as described here:

https://superuser.com/questions/173165/what-does-the-rc-in-bashrc-etc-mean

I've never seen it written "bash rc" before as opposed to "bashrc", and it made me curious what in the world "rc" actually meant -- I never knew, so I had to look it up.

Reply:
Neat! I didn't know this either and added it for a reference.